projects/demo/src/modules/directives/dropdown-context/examples/2/index.less

```diff
@@ -1,7 +1,23 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
 .context-menu {
     inline-size: 8rem;
 }
 
 .icon {
     border-width: 0.25rem;
 }
+
+[tuiTable] {
+    inline-size: 100%;
+}
+
+[tuiTh],
+[tuiTd] {
+    border-inline-start: none;
+    border-inline-end: none;
+}
+
+[tuiTd] {
```

Comment:
Let's also remove `border-block-start` form `[tuiTh]` so it looks more like it used to.

Reply:
Yeah, header border is appearing

Reply:
Resolved!